README.md

```diff
@@ -69,8 +69,12 @@ Flags include:
 --platform|-p <name>
 --arch|-a <name>
 --device|-d <name>
+--low-power
+--doze
 ```
 
+`--low-power` and `--doze` flags are specific to Android.
+
```

Comment:
I wonder if we could not use a markdown tables with these columns:
- Flag
- Description
- Notes

Reply:
It's currently copied from `bare-run --help` which is nice and simple. The flag descriptions aren't included, but we could add those so it'd look like this:

```
--version|-v           Print the current version
--base <path>          The base path of the bundle
--platform|-p <name>   The operating system platform to bundle for
--arch|-a <name>       The operating system architecture to bundle for
--device|-d <name>     The name of the device to launch
--low-power            Set phone to Low Power conditions, Android only
--doze                 Set phone to Doze mode, Android only
--help|-h              Show help
```